### PR TITLE
[Malbeacon] Modify Malbeacon connector to be playbook compatible

### PR DESCRIPTION
**Pull request by helene-nguyen**
### Proposed changes

* Convert API calls into STIX objects to be ingested 
* Add config variable for score
* Complete config variables for docker-compose.yml file
* Add types and comments

### Related issues

* https://github.com/OpenCTI-Platform/connectors/issues/1585

### Checklist

- [x] I consider the submitted work as finished
- [x] I tested the code for its functionality using different use cases
- [ ] I added/update the relevant documentation (either on github or on notion)
- [x] Where necessary I refactored code to improve the overall quality

### CODE REVIEW

File: internal-enrichment/malbeacon/src/malbeacon.py [hunk 1]
```diff
@@ -1,179 +1,593 @@
...
+                return info_msg
+            else:
+                info_msg = "[API] No information found on Malbeacon"
+                return info_msg
+
+    def _process_message(self, data: dict) -> str:
+        """
+        Get observable newly created from OpenCTI and process enrichment
+        :param data: Dictionary of data
+        :return: None
```
Comment (Megafredo) [inline internal-enrichment/malbeacon/src/malbeacon.py]: Maybe :return: str ? ^^

File: internal-enrichment/malbeacon/src/malbeacon.py [hunk 2]
```diff
@@ -1,179 +1,593 @@
...
+        is_valid_tlp = self.extract_and_check_markings(observable)
+        if not is_valid_tlp:
+            raise ValueError(
+                "[CONNECTOR] Do not send any data, TLP of the observable is greater than MAX TLP,"
+                "the connector does not has access to this observable, please check the group of the connector user"
+            )
 
-    def _process_observable(self, observable) -> str:
-        logger.info(f"processing observable: {observable}")
         # Extract IPv4, IPv6, Hostname and Domain from entity data
```
Comment (Megafredo) [inline internal-enrichment/malbeacon/src/malbeacon.py]: Scope include  Hostname ?

Comment (helene-nguyen) [reply internal-enrichment/malbeacon/src/malbeacon.py]: Removed from comment, not included yet

File: internal-enrichment/malbeacon/src/malbeacon.py (hunk 1 above)
Comment (helene-nguyen) [reply internal-enrichment/malbeacon/src/malbeacon.py]: Done, docstring corrected

**Review by Megafredo — APPROVED** [review]

**Pull request merged**